pkg/orchestrator/util.go

```diff
@@ -64,6 +64,8 @@ const (
 	K8sCR
 	// K8sVerticalPodAutoscaler represents a Kubernetes VerticalPod Autoscaler
 	K8sVerticalPodAutoscaler
+	// K8sHorizontalPodAutoscaler represents a Kubernetes Horizontal Pod Autoscaler
+	K8sHorizontalPodAutoscaler
```

Comment:
You also need to add hpa here https://github.com/DataDog/datadog-agent/blob/33ce9c2a8f7a161a292564b6b7e2738b322bf583/pkg/orchestrator/util.go#L95 and other functions